sources/packages/web/src/types/contracts/institution/EducationProgramDto.ts

```diff
@@ -143,6 +143,8 @@ export interface ProgramDto extends EducationProgramBaseDto {
   hasIntlExchange?: string;
   intlExchangeProgramEligibility?: string;
   programDeclaration: boolean;
+  hasOfferings?: boolean;
```

Comment:
Like API would recommend creating a separate DTO for GetProgram which has hasOfferings.